data/json/items/magazine/40mm.json

```diff
@@ -0,0 +1,11 @@
+[
+  {
+    "id": "belt40mm",
```

Comment:
Done. Had to remove copying stats from ammo belt class because "delete" field doesn't seem to delete "armor_data" field.